src/victory-primitives/clip-path.js

```diff
@@ -28,28 +29,25 @@ export default class ClipPath extends React.Component {
     translateY: PropTypes.number
   };
 
+  static defaultProps = {
+    translateX: 0,
+    translateY: 0
+  }
+
   componentWillMount() {
-    const { x, y, width, height } = this.calculateAttributes(this.props);
-    this.x = x;
-    this.y = y;
-    this.width = width;
-    this.height = height;
+    merge(this, this.calculateAttributes(this.props));
```

Comment:
Do you need `merge` here? I think these are all single value properties (non-nested).